release-notes/aws/v9.0.1.md

```diff
@@ -30,3 +30,7 @@ Below, you can find more details on components that were changed with this relea
   - Number of NGINX worker processes changed from 4 to 1
 - Updated from upstream `ingress-nginx` v0.29.0 - for details see the [changelog](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.30.0).
 - Configured app icon.
+
+### coredns v1.6.5 ([Giant Swarm app v1.1.7](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v117-2020-03-19))
+
+- Changed `autopath` value to disabled by default.
```

Comment:
Please consider also covering other coredns changes for completeness; release notes covering upgrade from aws 9.0.0 to 9.0.1 should probably mention all changes since (including) coredns-app 1.0.1 https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v110, because aws 9.0.0 it seems included pre 1.0.0 coredns-app.

You can safely include all changes that 9.1.0 release notes covered for coredns https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.1.0.md#coredns-v165-giant-swarm-app-v110
+ anything newer from coredns-app changelog